cmd/neofs-lens/internal/objects/root.go

```diff
@@ -0,0 +1,21 @@
+package objects
+
+import "github.com/spf13/cobra"
+
+var vPath string
+
+// Root defines root command for operations with NeoFS objects.
+var Root = &cobra.Command{
+	Use:   "objects",
```

Comment:
Why plural?

Reply:
planned to have `neofs-lens objects link`, `neofs-lens objects storage-group`, `neofs-lens objects lock`, etc. it should be either `objects` or `inspect-object` to me. your suggestion?

Reply:
`object`. `ip route`/`ip address`/etc. Ideally commands are verbs, but it's not the case already and you're always working with a single object here.

Reply:
ok, let it be `object`. i would go `objects` as a root command that works with different kinds of objects